File: src/Goods.Domain/Students/StudentBlank.cs
using Students.Domain.Enums;

namespace Students.Domain.Students;

public class StudentBlank
{
    public Guid? Id { get; set; }
    public String? FullName { get; set; }
    public Gender? Gender { get; set; }
    public Int32? Age { get; set; }
    public Single? AverageGrade { get; set; }
    public String[]? SpecialMarks { get; set; }
    public Guid? StudentGroupId { get; set; }

}

File: src/Students.BackOffice/client-app/src/domain/Students/StudentBlank.ts
import { Student } from './Student';
import { Gender } from './Gender';

export class StudentBlank {
	constructor(
		public id: string | null,
		public fullName: string | null,
		public gender: Gender | null,
		public age: number | null,
		public averageGrade: number | null,
		public specialMarks: string[] | null,
		public studentGroupId: string | null
	) { }
}

export namespace StudentBlank {
	export function getEmpty(): StudentBlank {
		return new StudentBlank(null, null, null, null, null, null, null);
	}

	export function getFromStudent(student: Student): StudentBlank {
		return new StudentBlank(
			student.id,
			student.fullName,
			student.gender,
			student.age,
			student.averageGrade,
			student.specialMarks,
			student.studentGroupId,
		)
	}
}
